examples/flutter_gallery/lib/demo/material/banner_demo.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../gallery/demo.dart';
+
+enum BannerDemoAction {
+  reset,
+  showMultipleActions,
+  showLeading,
+}
+
+class BannerDemo extends StatefulWidget {
+  const BannerDemo({ Key key }) : super(key: key);
+
+  static const String routeName = '/material/banner';
+
+  @override
+  _BannerDemoState createState() => _BannerDemoState();
+}
+
+class _BannerDemoState extends State<BannerDemo> {
+  bool displayBanner = true;
+  bool showMultipleActions = true;
+  bool showLeading = true;
+
+  @override
+  void initState() {
+    super.initState();
+    initBanner();
+  }
+
+  void initBanner() {
+    displayBanner = true;
+    showMultipleActions = true;
+    showLeading = true;
+  }
+
+  void handleDemoAction(BannerDemoAction action) {
+    setState(() {
+      switch (action) {
+        case BannerDemoAction.reset:
+          initBanner();
+          break;
+        case BannerDemoAction.showMultipleActions:
+          showMultipleActions = !showMultipleActions;
+          break;
+        case BannerDemoAction.showLeading:
+          showLeading = !showLeading;
+          break;
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final Widget banner = MaterialBannerTheme(child: MaterialBanner(
+      content: const Text('Your password was updated on your other device. Please sign in again.'),
+      leading: showLeading ? CircleAvatar(child: Icon(Icons.access_alarm)) : null,
+      actions: <Widget>[
+        FlatButton(
+          child: const Text('SIGN IN'),
+          onPressed: () =>
+            setState(() {
+              displayBanner = false;
+            })
+        ),
+        if (showMultipleActions)
+          FlatButton(
+            child: const Text('DISMISS'),
```

Comment:
s/DISMISS/Dismiss ?

Reply:
the Material guidelines recommend all caps for buttons

https://material.io/design/components/buttons.html#usage